code/obj/table.dm

```diff
@@ -516,6 +516,15 @@
 				actions.start(new /datum/action/bar/icon/fold_folding_table(src, null), user)
 		return
 
+/obj/table/syndicate
+	name = "crimson glass table"
+	desc = "An industrial grade table with a crimson glass panel on the top. The glass looks extremely sturdy."
+	icon = 'icons/obj/furniture/table_syndicate.dmi'
+	auto_type = /obj/table/syndicate
+	parts_type = /obj/item/furniture_parts/table/syndicate
+
+	auto
+		auto = 1
```

Comment:
```suggestion
		auto = TRUE
```

Reply:
Is there any real reason for this? All other tables use 1 or 0 instead of TRUE or FALSE.

Reply:
You shouldn't be using 1/0 instead of TRUE/FALSE, may as well fix it versus adding to the issue.

Besides, TRUE/FALSE are preprocessor arguments, the compiler reads it as 1/0